Add new berets for XO

potatoed6 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
This PR adds new berets - Green/Tan XO beret in a vendor, Ocean-wave XO beret for 50 XO hours and Black XO beret for 120 XO hours.

## Why / Balance
XO's being first or second in command, lack unique style, looking as SO. With this change, XO's will be able to change their appearance to more unique.

## Technical details
Added .png's, yml's, edited required yml for vendors, groups, etc.

## Media
<img width="279" height="191" alt="image" src="https://github.com/user-attachments/assets/039ec0fd-e9e4-4b28-b4b9-39af3391da5d" />
<img width="215" height="275" alt="image" src="https://github.com/user-attachments/assets/9c81cc8a-6daa-4a37-b2db-03cb7b9e69d1" />
<img width="264" height="229" alt="image" src="https://github.com/user-attachments/assets/a7d62d21-f0fc-46fb-8023-5307ba219e83" />
<img width="192" height="296" alt="image" src="https://github.com/user-attachments/assets/9e15a3d3-bc6b-4267-8c01-83cb9600258a" />
<img width="107" height="190" alt="image" src="https://github.com/user-attachments/assets/0ba58e76-4ae8-4e46-9765-29de08292b58" />
<img width="156" height="98" alt="image" src="https://github.com/user-attachments/assets/557fdf80-6022-4385-a4c3-0462cfd1f3bb" />
<img width="105" height="172" alt="image" src="https://github.com/user-attachments/assets/12efe8c6-48da-4f9d-9522-e216fd6dafdf" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->
- [x] By submitting this code and/or assets, I confirm that I either own them or have provided the correct necessary licenses to use and distribute them. I agree to be fully responsible for any legal claims or issues arising from the use of these materials.

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: Added XO's tan/green beret, XO's ocean-wave beret for 50 XO hours and XO's black beret for 120 XO hours.
- add: Added XO's role-specific category in loadout - for berets.
- tweak: Changed normal beret from XO's equipment vendor to XO's green/tan beret. Added officer beret.

potatoed6 committed:
uniform, berets

added uniform and berets,
modified loadout and vendor

potatoed6 committed:
changed textures, added beret to vend, cleared debug

potatoed6 committed:
new berets

potatoed6 committed:
updated prime beret textures

potatoed6 committed:
fixed classic uniform

chavonadelal reviewed (commented):
You forgot to add a changelog about your changes.

potatoed6 commented:
grrr, changelog misteriously gone

QuietlyWhisper commented:
something is off about these sprites but I'm not sure what it is, might need to see side-by-side with existing uniforms... ask spriting channel for opinions maybe

potatoed6 commented:
i think you might be talking about outlines, if it is about uniforms.

I'll try modify it

OmerF-afk commented:
<img width="199" height="51" alt="image" src="https://github.com/user-attachments/assets/0157caeb-d498-4d9b-9bdf-6215da48506e" />
The black beret seems to be missing some black outline lines. (and cyan one has extra on top of it that goes above outline)
Grey one is cool, but it feels so flat next to the other ones.

Graded5 commented:
HOLY SOUL???

potatoed6 commented:
<img width="351" height="317" alt="image" src="https://github.com/user-attachments/assets/65f564cd-9fb9-4fb6-9dc9-e15276fb8f75" />
<img width="327" height="369" alt="image" src="https://github.com/user-attachments/assets/848d6f36-cf52-42de-a2c2-26d5f55ba0a3" />
<img width="400" height="344" alt="image" src="https://github.com/user-attachments/assets/27e5dfd8-df08-4d77-9b66-164fbc3240c3" />
Updated uniform outlines, tan beret texture, fixed beret icons

potatoed6 committed:
Fixes

 Updated uniform outlines, tan beret texture, fixed beret icons

potatoed6 commented:
Those are from dummies that are used in med

PursuitInAshes reviewed (changes requested):
Just a basic review. I do not know how the hosts will feel about the playtime hours specifically, so they might ask you to change that later.

Something is off with the sprites, but I am unsure how best to go about fixing them. I think its a combination of the way the outlining is done and the noise on them feeling strange. I'd suggest putting them side by side the standard service uniform sprites to get a better reference on how they could look and the art style generally. I think specifically on the uniforms the front pockets feel strange and the back sprite feels off.
You are also more than free to ask for guidance in the sprite channel under the development category on the discord.

potatoed6 committed:
Merge branch 'master' into pr/7923

potatoed6 committed:
Update hats.yml

potatoed6 committed:
deleting XO camo, fixing errors

potatoed6 committed:
fixing copyrights and adding Officer beret to the XO

BramvanZijp commented:
@potatoed6 I will note, for CO, the playtime-locked berets cost 0, I feel like having this also apply to the XO playtime-locked berets might be nice, so that they can display their playtime with said beret without needing to sacrifice other loadout items.

potatoed6 committed:
Update executive_officer.yml

potatoed6 commented:
thats fair, updated

potatoed6 committed:
Merge branch 'master' into pr/7923

potatoed6 committed:
Merge branch 'master' into pr/7923

PursuitInAshes commented on Resources/Prototypes/_RMC14/Roles/Loadouts/Items/RoleSpecific/executive_officer.yml:
A development from when I last reviewed this, but you'll want to set the unlock times for these inbetween medals instead of at them. It was decided that its generally preferred playtime rewards like these happen inbetween medals

I'd suggest something inbetween silver/gold and gold/plat

(CO berets were an exception to this due to them corresponding to direct rankups)

PursuitInAshes reviewed (changes requested):
Update Media on the PR itself just for ease of review from others.
This is just YAML, you'll need to wait for someone to look at sprite and meta

potatoed6 committed:
Update executive_officer.yml

potatoed6 committed:
Update executive_officer.yml

PursuitInAshes reviewed (approved):
Code good
Sprites/Meta will need checked by someone else though

potatoed6 committed:
Merge branch 'master' into pr/7923

potatoed6 committed:
Merge branch 'master' into pr/7923

potatoed6 committed:
Merge branch 'master' into xo-new-uniform

potatoed6 committed:
Update hats.yml
